test/verify/run_verify.cpp

```diff
@@ -188,7 +188,8 @@ auto get_hash(const T& x)
 
 void run_verify::verify(const std::string& name,
                         const migraphx::program& p,
-                        const migraphx::compile_options& c_opts) const
+                        const migraphx::compile_options& c_opts,
+                        size_t tolerance) const
```

Comment:
We should probably pass `program_info` struct directly to this method instead of so many parameters.

Reply:
Done.